src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -841,7 +881,7 @@ private static bool IsPrintableStringCharacter(char c)
             return false;
         }
 
-        private static byte[] ConcatenateArrays(byte[][][] segments)
+        private static byte[] ConcatenateArrays(params byte[][][] segments)
```

Comment:
I see.  And we have other existing call sites that are already passing in more than one array? Ok.